app/containers/filterContainer/yearRangeSlider/index.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from "react";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+import * as classNames from "classnames";
+import { withStyles } from "../../../helpers/withStylesHelper";
+import { MIN_YEAR } from "./constants";
+import Slider from "./slider";
+import { SearchPageQueryParams } from "../../../components/articleSearch/types";
+import getQueryParamsObject from "../../../helpers/getQueryParamsObject";
+import PapersQueryFormatter from "../../../helpers/papersQueryFormatter";
+import { goToYearFilteredSearchResultPage, GoToYearFilteredSearchResultPageParams } from "./helper";
+import FilterButton from "../filterButton";
+const styles = require("./yearRangeSlider.scss");
+
+interface YearSet {
+  year: number;
+  docCount: number;
+}
+
+interface YearRangeSliderProps extends RouteComponentProps<null> {
+  yearInfo: YearSet[];
+  filteredYearInfo: YearSet[];
+}
+
+const Column: React.FunctionComponent<{
+  width: string;
+  height: string;
+  active: boolean;
+  yearSet: YearSet;
+  isSelecting: boolean;
+  isSelectMode: boolean;
+  filteredYearSet: YearSet | undefined;
+  filterHeight: string;
+  destinationParams: GoToYearFilteredSearchResultPageParams;
+}> = React.memo(props => {
+  return (
+    <>
+      {props.filteredYearSet &&
+        props.filteredYearSet.docCount > 0 && (
+          <div
+            style={{
+              width: props.width,
+              height: props.filterHeight,
+              backgroundColor: "#3e7fff",
+              borderRight: "solid 1px #f9f9fa",
```

Comment:
It seems that you can separate these style property values by declaring them as variables.

Reply:
Not necessary.